psi4/driver/procrouting/dft_funcs/dict_hyb_funcs.py

```diff
@@ -0,0 +1,494 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2017 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""
+List of hybrid functionals
+"""
+
+import copy
+
+funcs = []
+
+
```

Comment:
Can we keep it consistent with YAPF enable/disable blocks rather than for every function?

Reply:
What happens is that sometimes yapf prefers the `"name": "method_name"` on the same line, sometimes it splits them onto newline. The same for `"citation"` and `"description"`. If you're not fussed about this, I can just remove the yapf disables...